src/plugins/googledrive/admin/src/api/delDelete.js

```diff
@@ -0,0 +1,35 @@
+import axiosInstance from '../utils/axiosInst';
+
+export const delDelete = async (infileId) => {
+  try {
+    const delMessage = await axiosInstance.request(
+      '/googledrive/delete?fileId=' + infileId,
+      {
+        headers: { Authorization: `Bearer ${process.env.ADMIN_JWT_SECRET}` },
+      },
+      {
+        method: 'delete',
+      }
+    );
+    return delMessage.status;
+  } catch (err) {
+    return err;
+  }
+};
+export const postUpdate = async (inUploadData) => {
```

Comment:
The file name is `delDelete.js`. Should an update function be in another folder?